com.unity.robotics.urdf-importer/Tests/Runtime/UrdfComponents/UrdfJoints/UrdfJointTests.cs

```diff
@@ -0,0 +1,303 @@
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using RosSharp.Urdf;
+
+public class TestUrdfJoint : UrdfJoint
+{
+    public override JointTypes JointType => throw new System.NotImplementedException();
+
+    public static Vector3 Axis(RosSharp.Urdf.Joint.Axis axis) => GetAxis(axis);
+    public static Vector3 DefaultAxis() => GetDefaultAxis();
+    public static RosSharp.Urdf.Joint.Axis AxisData(Vector3 axis) => GetAxisData(axis);
+    public void Dynamics(RosSharp.Urdf.Joint.Dynamics dynamics) => SetDynamics(dynamics);
+}
+
+public class UrdfJointTests
+{
+#if UNITY_2020_1_OR_NEWER
+    [Test]
+    public void Create_FixedArticulationBody()
+    {
+        GameObject linkObject = new GameObject("link");
+        UrdfJoint.Create(linkObject, UrdfJoint.JointTypes.Fixed);
```

Comment:
nit: Do you think the Create method should return UrdfJoint instead of void?

Reply:
Good point! I just modified it and made it return a UrdfJoint